docs/content-services/components/search-filter-tabbed.component.md

```diff
@@ -0,0 +1,61 @@
+Status: Active
+Last reviewed: 2023-06-28
+---
+
+# [Search filter tabbed component](../../../lib/content-services/src/lib/search/components/search-filter-tabbed/search-filter-tabbed.component.ts "Defined in search-filter-tabbed.component.ts")
+
+Represents a container for [Search Filters](search-filter.component.md) to provide a tabbed user interface for the filters.
+
+![Search Filter Tabbed Widget](../../docassets/images/search-filter-tabbed.png)
+
+## Basic Usage
+
+```html
+
+<adf-search-filter-tabbed>
+    <ng-container *ngFor="let field of fields">
+        <my-search-filter *adf-search-filter-tab="MyTabLabel"></my-search-filter>
+    </ng-container>
+</adf-search-filter-tabbed>
+```
+
+In order to generate a tabbed widget for multiple search filters, you can pass in the search filter widget component as a content child of the adf-search-filter-tabbed component as shown above.
+
+Additionally, you also have to make sure that the search filter being passed as a content child of the adf-search-filter-tabbed component, also has the adf-search-filter-tabbed directive applied on it,
+with the name input property being assigned the value of whatever name should be displayed for that particular tab
+
+## Class Members
+
+### Properties
+
+| Name            | Type                                                                                                            | Description                                                                                                                                                                                                   |
+|-----------------|-----------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| settings        | [SearchWidgetSettings](../../../lib/content-services/src/lib/search/models/search-widget-settings.interface.ts) | The settings object for the search filter widget. This is mainly used to evaluate the fields and display labels for the different tabs.                                                                       |
+| queries         | { [key: string]: string }                                                                                       | A Javascript object containing all the queries for the different tabs, identified by the fields of the tabs. This is used to generate a single query by combining all the provided queries using their fields |
+| valuesToDisplay | { [key: string]: string }                                                                                       | A Javascript object containing the display values for all the different tabs. This is used to generate a single display label for all the tabs.                                                               |
+
+### Events
```

Comment:
All these events seem don't exist anymore

Reply:
These have been removed in [this](https://github.com/Alfresco/alfresco-ng2-components/pull/8714/commits/bdaa0e52fd3a31ba7c3802c6cdc6a27a733d6057) commit